src/components/common/vatsim/CommonEventCard.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+    <div>
+        <div
+            class="event-card"
+            :class="{ 'active-event': active }"
+            @click="details = !details"
+        >
+            <div class="event-card_start">
+                {{ formattedStart }}Z - {{ formattedEnd }}Z
+            </div>
+            <div class="event-card_start">
+                {{ props.event.organisers[0]?.region }} -> {{ props.event.organisers[0]?.division }}
+            </div>
+            <div class="event-card_name">
+                {{ props.event.name }} <span
+                    v-if="active"
+                    class="active"
```

Comment:
```suggestion
                    class="event-card__active"
```

or event-card_name_active